src/components/PopupWithForm.js

```diff
@@ -4,6 +4,7 @@ export default class PopupWithForm extends Popup {
     super({ popup });
     this._popupForm = this._popup.querySelector(".form");
     this._formSubmit = formSubmit;
+    this.submitButton = this._popupForm.querySelector(".form__button");
   }
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`